x/gov/README.md

```diff
@@ -597,12 +643,31 @@ The `constitution` string will be updated by applying the patch defined in the `
 An error will be returned if the `amendment` string is malformed, so constitution amendment proposals
 need to be crafted with care.
 
+### Last Min Deposit and Last Min Initial Deposit
+
+The `LastMinDeposit` and `LastMinInitialDeposit` are used to store the last values
+of the dynamic `MinDeposit` and `MinInitialDeposit` respectively upon proposals
+activation or deactivation. These values are used to determine the current values
+for the dynamic `MinDeposit` and `MinInitialDeposit` accounting also for the
+passage of time as detailed in [ADR-003](../../docs/architecture/adr-003-governance-proposal-deposit-auto-throttler.md)
+
+**Store:**
+
+```protobuf reference
+https://github.com/atomone-hub/atomone/blob/fb05dcaba40c7a1531a6806487fcd47a3e4aaef4/proto/atomone/gov/v1/gov.proto#L51-L60
```

Comment:
we might have to update this and the other protobuf references after we merge this PR